persistence-modules/spring-data-redis/src/main/java/com/baeldung/spring/data/redis_ttl/repository/BookRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.spring.data.redis_ttl.repository;
+
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Repository;
+
+import com.baeldung.spring.data.redis_ttl.entity.Book;
+
+@Repository
+public class BookRepository{
+
+    private RedisTemplate<String, Book> redisTemplate;
+    
+    @Autowired
+    public BookRepository(RedisTemplate<String, Book> redisTemplate) {
+      this.redisTemplate = redisTemplate;
+    }
+
+
+    public Book get(String title) {
+        return redisTemplate.opsForValue().get(title);
+    }
+    
+    public void save(Book book) {
+        redisTemplate.opsForValue().set(book.getTitle(), book);
+        
+        int ttlValue = 5; //secs 
+        if(book.getLanguage().toLowerCase().equals("english"))
+            ttlValue = 10;
```

Comment:
missing `{}` around if statement's expression

Reply:
Java does allow unbraced if stmts, right ? So, is the comment more from a styling perspective?